flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNodeBase.java

```diff
@@ -276,4 +277,9 @@ protected String createFormattedTransformationName(
     public void resetTransformation() {
         this.transformation = null;
     }
+
+    @VisibleForTesting
+    public boolean isTransformationNull() {
```

Comment:
just provide getTransformation method for testing

Reply:
> just provide getTransformation method for testing

Done!